blockmanager.go

```diff
@@ -1341,14 +1310,14 @@ func newBlockManager(s *server) (*blockManager, error) {
 	}
 
 	bm := blockManager{
-		server:           s,
-		requestedTxns:    make(map[btcwire.ShaHash]struct{}),
-		requestedBlocks:  make(map[btcwire.ShaHash]struct{}),
-		lastBlockLogTime: time.Now(),
-		msgChan:          make(chan interface{}, cfg.MaxPeers*3),
-		headerList:       list.New(),
-		quit:             make(chan struct{}),
-	}
+		server:          s,
+		requestedTxns:   make(map[btcwire.ShaHash]struct{}),
+		requestedBlocks: make(map[btcwire.ShaHash]struct{}),
+		msgChan:         make(chan interface{}, cfg.MaxPeers*3),
+		headerList:      list.New(),
+		quit:            make(chan struct{}),
+	}
+	bm.progressLogger = newBlockProgressLogger("Processed")
```

Comment:
This can created inside the initialization struct since it doesn't require a reference to the block manager itself.